node-api/handlers/proof/merkle/proposer_pubkey_test.go

```diff
@@ -69,30 +70,37 @@ func TestBlockProposerPubkeyProof(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		t.Run(tc.name, func(t *testing.T) {
-			vals := make(types.Validators, tc.numValidators)
-			for i := range vals {
-				vals[i] = &types.Validator{}
-			}
-			vals[tc.proposerIndex] = &types.Validator{Pubkey: tc.pubKey}
+		for _, forkVersion := range version.GetSupportedVersions() {
```

Comment:
need to remove the change to iterate through supported versions. These tests are based on pre-generated proofs in testdata folder, so each testcase will have its own version to specify (cannot run all fork versions on each testcase. https://github.com/berachain/beacon-kit/pull/2661 will add the new testcases

Reply:
resolved in https://github.com/berachain/beacon-kit/pull/2656/commits/d8a4bbe7ea3ba682ff5e048120a197763967be7d